### PR TITLE
Using the save and next icon

### PR DESCRIPTION
**Pull request by rustedgrail**
fixes #

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

File: app/react/UI/Icon/save-and-next.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ * </svg>
+ */
+
+const saveAndNextSvgPath = "M6 5.5V0.5C6 0.223858 5.77614 0 5.5 0H1.5C0.671573 0 0 0.671573 0 1.5V18.5C0 19.3284 0.671573 20 1.5 20H18.5C19.3284 20 20 19.3284 20 18.5V4.62132C20 4.2235 19.842 3.84196 19.5607 3.56066L16.4393 0.43934C16.158 0.158035 15.7765 0 15.3787 0H14.5C14.2239 0 14 0.223858 14 0.5V5.5C14 5.77614 13.7761 6 13.5 6H6.5C6.22386 6 6 5.77614 6 5.5ZM11.5 0H10.5C10.2239 0 10 0.223858 10 0.5V3.5C10 3.77614 10.2239 4 10.5 4H11.5C11.7761 4 12 3.77614 12 3.5V0.5C12 0.223858 11.7761 0 11.5 0ZM9.29289 9.70718L11.5858 12.0001H6V14.0001H11.5858L9.29289 16.293L10.7071 17.7072L15.4142 13.0001L10.7071 8.29297L9.29289 9.70718Z";
+export const saveAndNext = {
+    prefix: "fas",
+    iconName : "save-and-next",
+    icon: [20, 20, [], 'e001', saveAndNextSvgPath]
+};
```
Comment (bdittes) [inline app/react/UI/Icon/save-and-next.js]: newline (maybe auto-format the file?)

File: app/react/UI/Icon/library.js [hunk 2]
```diff
@@ -175,10 +176,13 @@ const icons = {
...
   faUserTimes,
   faHandPaper,
+  saveAndNext,
 };
 
 export const loadIcons = () => {
   library.add(...Object.keys(icons).map(key => icons[key]));
 };
 
+dom.watch();
```
Comment (bdittes) [inline app/react/UI/Icon/library.js]: comment to doc?

Comment (RafaPolit) [reply app/react/UI/Icon/library.js]: Why is this  required and why was it not needed before? Thanks.

Comment (rustedgrail) [reply app/react/UI/Icon/library.js]: Looks like it isn't required. It was in the instructions to add a custom icon, but those weren't react specific instructions. I'm guessing that the component already calls this.

**Pull request merged**